### ISSUE TITLE
[feature request] More pictures upload

### ISSUE THREAD

**Issue by copiis** [jasoncheng/dica#83]
It would be nice if more than one picture can be uploaded when creating a new post.

**Comment by jasoncheng** [comment]
It should be few hours job on Android develop, but API support multiple IDs ?

<img width="969" alt="2018-12-12 12 50 02" src="https://user-images.githubusercontent.com/540463/49816007-f36a5d00-fda7-11e8-95d8-15a702f81c33.png">

**Comment by copiis** [comment]
I do not know that.
Through the web interface, the images must first be uploaded to the server and then they will paste, maybe that's a problem in the app

**Comment by jasoncheng** [comment]
i also want can upload movie for you @copiis 

<img width="989" alt="2018-12-12 12 55 40" src="https://user-images.githubusercontent.com/540463/49816377-ae92f600-fda8-11e8-8aba-257cfe48094d.png">

**Comment by copiis** [comment]
thet would be GREAT

**Comment by jasoncheng** [comment]
Yes, API not support yet; But actually i can to the tricky way; 
upload and use bbcode [img] to attach, should i do that?

**Comment by copiis** [comment]
you have a GO from me! Let's see what the post looks like when it is shared.

**Comment by jasoncheng** [comment]
okay, after tomorrow, let's do this!
maybe use [img] is a best solution instead of empty attachments

**Comment by copiis** [comment]
Hmm, would it be possible to first upload the pictures in a row and then publish the post with the two pictures?

**Comment by jasoncheng** [comment]
you mean publish two status at same time?
1. first photo status
2. a status with photos

**Comment by copiis** [comment]
No, first upload the photos in the background in a row and then publish the status with the two or three photos in one post.

**Comment by jasoncheng** [comment]
got it 👍

**Comment by jasoncheng** [comment]
status update:  @copiis 

after research media/upload API, it's doesn't return image URL for later use;

`{"media_id":542219,"media_id_string":"542219","size":257439,"image":{"w":2400,"h":2400,"image_type":"image\/jpeg"}}`

So, isn't possible for now, should looking another solution, may upload to user album, use friendica/photo/update(create)

**Comment by jasoncheng** [comment]
status update: 
fail: after digging Friendica API source code again, i cannot generate url even i have photo id
no way to do on current API stage

system::createGUID(32, false)
$photo_hash = Photo::newResource()
/photo/{$photo_hash}-{$smallest}

:{

**Comment by jasoncheng** [comment]
okay, i decide do something on it @copiis 
https://github.com/friendica/friendica/issues/6272
